Update Contact Us Section, added new Section (#95)
pulsar-edit/.github: FAQ.md

# How do I contact members of Pulsar?

There are many ways, of course the classical Issues or Pull Requests on Repos is fine and well, additionally there are [discussions](https://github.com/orgs/pulsar-edit/discussions) org wide if needed. We are also all pretty active and responsive on our [Discord Server](https://discord.gg/7aEbB9dGRT), and messages there are always welcome. Additionally feel free to contact us through any of the below resources, or check out our [Org Wide Readme](https://github.com/pulsar-edit) as it's most up to date.

* [GitHub Discussions](https://github.com/orgs/pulsar-edit/discussions)
* [Discord Server](https://discord.gg/7aEbB9dGRT)
* [Reddit](https://www.reddit.com/r/pulsaredit/)
* [Mastodon](https://fosstodon.org/@pulsaredit)

# Why don't you rewrite Pulsar in **insert Popular Web App Framework or Language Here**

At this time, here is our current stance:

> Firstly, moving away from Node and Electron will mean a compelete rewrite of the whole editor, something we do not have the manpower to do.

Reviewer: ```suggestion
Firstly, moving away from Node and Electron would mean a complete rewrite of the whole editor, something we do not have the manpower to do.
```

sp.

> Additionally moving away from these technologies will break every single package we have. The ecosystem of Atom and it's community packages is a huge reason we wanted to keep it alive, rendering these completely useless removes one of the biggest advantages of the editor as a whole, and asking every single package author to rewrite their packages is not realistic.

Reviewer: ```suggestion
Additionally moving away from these technologies will break every single package we have. The ecosystem of Atom and its community packages is a huge reason we wanted to keep it alive, rendering these completely useless removes one of the biggest advantages of the editor as a whole, and asking every single package author to rewrite their packages is not realistic.
```

sp.

> Even all of our own packages will need to be rewritten, and all of our Native Packages, which are already some of the most combersome to maintain, will no longer function in this new environment.

Reviewer: ```suggestion
Our own packages would have to be rewritten along with all of our Native Packages (which are already some of the most cumbersome to maintain) which would no longer function in this new environment.
```

> Lastly, Atom is the editor that created Electron, while it may have it's idiosyncrasies, we feel that it seems almost wrong to ditch it for some hot new thing.

Reviewer 2: 💯 

I never personally answered this question when it was brought up, but I feel that this is the single most important thing of all of the reasons. We want to keep the legacy alive, and that includes Electron, even though its quite a... lets just say controversial technology.

Author: Totally agree, it's not fancy and new, but it's what created our editor, and should be used until it truly only makes sense to switch. Not because there's something newer.

Reviewer: This has indeed been my primary stance on it, Atom made Electron so removing it literally removes a huge chunk of what makes Atom, Atom.
Electron isn't fundamentally broken, a Javascript/web technology app is never going to be as fast as something lightweight written in C/C++/Rust but that isn't its goal and it was never Atom's goal.

> Lastly, Atom is the editor that created Electron, while it may have it's idiosyncrasies, we feel that it seems almost wrong to ditch it for some hot new thing.

Reviewer: ```suggestion
Lastly, Atom is the editor that created Electron, while it may have its idiosyncrasies, we feel that it seems almost wrong to ditch it for some hot new thing.
```
sp.

> The focus of our team as seen by our mission statement is to modernize and improve Atom, bringing it to the 21st Century, and create a proud sucessor of Atom called Pulsar. It is not to create a brand new editor from scratch.

Reviewer: > bringing it to the 21st Century

Small gripe, makes it sounds like Atom was made in 1999 or something, "bring it up to date" or something sure but I think this almost sounds like a disservice to Atom.

Reviewer 3: Well there is the thing we gotta show we're trying to move from Electron 9 and node 12(?) to whatever the current versions are or get damn near close to it. Since I think both are moving into v20+ and we're just even trying to get caught up on that. Let alone removing any stuff dated to earlier node/electron
